Cubical/Data/Nat/Properties.agda

```diff
@@ -179,3 +180,32 @@ _choose_ : ℕ → ℕ → ℕ
 n choose zero = 1
 zero choose suc k = 0
 suc n choose suc k = n choose (suc k) + n choose k
+
+isEven'-suc : (n : ℕ) → isOdd' n → isEven' (suc n)
```

Comment:
Don't these hold by def for the other def of isEven/isOdd?

Reply:
Oh yeah, wups. For both definitions actually.

Reply:
I think I'll just remove this actually